sql/pre_install/bgw_startup.sql

```diff
@@ -0,0 +1,10 @@
+-- We have to create the force restart function here and run it in order to avoid a race condition
```

Comment:
I think the comment  can be more clear. The race condition is when a create extension start after the scheduler has been started but create extension commits after the scheduler checks for the existence of the extension.